drivers/storage/portworx/deployment_test.go

```diff
@@ -4436,3 +4436,105 @@ func assertContainerEqual(t *testing.T, expected, actual v1.Container) {
 	assert.ElementsMatch(t, expected.Env, actual.Env)
 	assert.ElementsMatch(t, expected.VolumeMounts, actual.VolumeMounts)
 }
+
+func Test_template_overrideVolumeMount(t *testing.T) {
+	testVols := getCommonVolumeList(pxutil.MinimumPxVersionAutoTLS)
+	hp0 := &v1.HostPathVolumeSource{
+		Path: "/test0",
+	}
+	overRideVols := append(testVols, []volumeInfo{}...)
+	for i := range overRideVols {
+		if overRideVols[i].name == "diagsdump" {
+			overRideVols[i].hostPath = hp0.String()
+			overRideVols[i].mountPropagation = mountPropagationModePtr(v1.MountPropagationBidirectional)
+			break
+		}
+	}
+
+	hp1 := &v1.HostPathVolumeSource{
+		Path: "/test1",
+	}
+	hp2 := &v1.HostPathVolumeSource{
+		Path: "/test1",
```

Comment:
shoul this be "/test2" ?

Reply:
wouldn't matter actually that's just a dummy name being used

Reply:
changed it thought